src/main/java/cuchaz/enigma/gui/dialog/JavadocDialog.java

```diff
@@ -87,4 +133,27 @@ public static void init(JFrame parent, JTextArea area, Callback callback) {
 	public interface Callback {
 		void closeUi(JFrame frame, boolean save);
 	}
+
+	private enum JavadocTag {
+		CODE(true),
+		LINK(true),
+		LINKPLAIN(true),
+		RETURN(false),
+		SEE(false),
+		THROWS(false);
```

Comment:
Note that custom tags exist in java, as there are taglets. 
https://docs.oracle.com/javase/8/docs/jdk/api/javadoc/taglet/com/sun/tools/doclets/Taglet.html

I actually hope that enigma can parse jd non-inline tags from raw comments and allow us to edit those accordingly than this simple tag-adding tool in gui.